Source/Scene/Cesium3DTile.js

```diff
@@ -61,30 +67,69 @@ define([
         var contentHeader = header.content;
 
 // TODO: For the 3D Tiles spec, we want to change this to basically (x, y, z) to (x, y, z)
-        var b = header.box;
-        var rectangle = new Rectangle(b[0], b[1], b[2], b[3]);
-
-        this._tileBoundingBox = new TileBoundingBox({
-            rectangle : rectangle,
-            minimumHeight : b[4],
-            maximumHeight : b[5]
-        });
-        this._orientedBoundingBox = OrientedBoundingBox.fromRectangle(rectangle, b[4], b[5]);
+        var tileBoundingVolume = header.boundingVolume;
+        if (defined(tileBoundingVolume.box)) {
+            var box = tileBoundingVolume.box;
+            var rectangleBox = new Rectangle(box[0], box[1], box[2], box[3]);
+
+            this._boundingVolume = new TileOrientedBoundingBox({
+                rectangle : rectangleBox,
+                minimumHeight : box[4],
+                maximumHeight : box[5]
+            });
+        } else if (defined(tileBoundingVolume.region)) {
+            var region = tileBoundingVolume.region;
+            var rectangleRegion = new Rectangle(region[0], region[1], region[2], region[3]);
+
+            this._boundingVolume = new TileBoundingRegion({
+                rectangle : rectangleRegion,
+                minimumHeight : region[4],
+                maximumHeight : region[5]
+            });
+        } else if (defined(tileBoundingVolume.sphere)) {
+            var sphere = tileBoundingVolume.sphere;
+
+            this._boundingVolume = new TileBoundingSphere(
+                new Cartesian3(sphere[0], sphere[1], sphere[2]),
+                sphere[3]
+            );
+        }
 // TODO: if the content type has pixel size, like points or billboards, the bounding volume needs
 // to dynamic size bigger like BillboardCollection and PointCollection
 
-        var rs;
-        if (defined(contentHeader) && defined(contentHeader.box)) {
+        var contentBoundingVolume;
+        if (defined(contentHeader) && defined(contentHeader.boundingVolume)) {
             // Non-leaf tiles may have a content-box bounding-volume, which is a tight-fit box
             // around only the models in the tile.  This box is useful for culling for rendering,
             // but not for culling for traversing the tree since it is not spatial coherence, i.e.,
             // since it only bounds models in the tile, not the entire tile, children may be
             // outside of this box.
-            var cb = contentHeader.box;
-            rs = OrientedBoundingBox.fromRectangle(new Rectangle(cb[0], cb[1], cb[2], cb[3]), cb[4], cb[5]);
+            var headerVolume = contentHeader.boundingVolume;
+
+            if (defined(headerVolume.box)) {
```

Comment:
This is basically duplicate with the `if` block above, right?  Can we introduce a helper that returns the bounding volume instance given the 3D Tiles `boundingVolume` property?